053-SentinelAutomatedResponse/Coach/Solution-03.md

```diff
@@ -0,0 +1,50 @@
+# Challenge-03 - Automated Response - Coach's Guide
+
+[< Previous Challenge](./Solution-02.md) - **[Home](./README.md)** 
+
+
+## Introduction
+In this section of the Coach's guide, you fill find guidance, code, and examples that will help you guide the teams over the course of the WTH. 
+In the spirit of continuous improvement, update this repository with any suggestions, altertnatives, or additional challenges.
+
+This section of the hack includes a optional [Sentinel Review Deck](SentinelWTHChallenge02.pptx) that features a short presentation to introduce key topics associated with 
+this challenge. 
+
+## Part1
+
+1. Install the following playbook andtest the connections.  
+https://github.com/Azure/Azure-Sentinel/tree/master/Playbooks/Watchlist-CloseIncidentKnownIPs </br>
+
+    When installing the logic app, you need to provide credentials.  For those using a Service Principle use the following: </br>
+    ID's shown below are for example only. </br>
+
+* Create a service principal by creating an “App Registration”  in my case I used Sentinel-Logic-App-ServicePrincipal.
```

Comment:
Please delete the special object character between "I used" and "Sentinel-Logic-App-ServicePrincipal"